components/Modal/ManageSources.js

```diff
@@ -356,100 +361,109 @@ class ManageSourcesModal extends React.Component {
-                          edited={this.state.editName}
-                          fetching={manageSources.fetchingSources}
-                          edit={this.handleEditSource}
-                          remove={this.props.removeSource}
-                        />
-                      ))}
-                  </div>
-                </>
-              )) ||
-                manageSourcesForm}
-            </>
-          )}
-        </Modal.Body>
-        <Modal.Footer>
-          {(!this.state.addEntry && (
-            <button type="button" className="btn btn-default" onClick={this.props.close}>
-              <FormattedMessage defaultMessage="Close" />
-            </button>
-          )) || (
-            <>
-              {manageSources.fetchingSources && (
-                <div className="pull-left">
-                  <Spinner loading size="xs" inline />
-                  <FormattedMessage defaultMessage="Saving source" />
                 </div>
-              )}
-              <button type="button" className="btn btn-default" onClick={(e) => this.handleShowForm(e, false)}>
-                {formatMessage(messages.cancel)}
-              </button>
+              </>
+            )) ||
+              manageSourcesForm}
+          </>
+        )}
+      </>
+    );
 
-              <button
-                type="submit"
-                className="btn btn-primary"
-                form="cmpsr-form-add-source"
-                disabled={disabledSubmit}
-                onClick={() => this.handleSubmitSource()}
-              >
-                {(this.state.editName === "" && formatMessage(messages.save)) || formatMessage(messages.update)}
-              </button>
-            </>
-          )}
-        </Modal.Footer>
+    const footer = (
+      <>
+        {(!this.state.addEntry && (
+          <button type="button" className="btn btn-default" onClick={this.props.close}>
+            <FormattedMessage defaultMessage="Close" />
+          </button>
+        )) || (
+          <>
+            {manageSources.fetchingSources && (
+              <div className="pull-left">
+                <Spinner loading size="xs" inline />
+                <FormattedMessage defaultMessage="Saving source" />
+              </div>
+            )}
+            <button type="button" className="btn btn-default" onClick={(ev) => this.handleShowForm(ev, false)}>
+              {formatMessage(messages.cancel)}
+            </button>
+            <button
+              type="submit"
+              className="btn btn-primary"
+              form="cmpsr-form-add-source"
+              disabled={disabledSubmit}
+              onClick={(ev) => this.handleSubmitSource(ev)}
+            >
+              {(this.state.editName === "" && formatMessage(messages.save)) || formatMessage(messages.update)}
+            </button>
+          </>
+        )}
+      </>
+    );
+
+    return (
+      <Modal
+        isOpen
+        variant={ModalVariant.medium}
```

Comment:
admittedly, I haven't run this, but we also specify `position="top"`alongside with `variant="medium"`. But it might be, that without this position it aligns more with other existing dialogs?

Reply:
Added the `position="top"`